test/tests/utils/spline_interpolation/tests

```diff
@@ -9,20 +9,17 @@
     input = 'bicubic_spline_interpolation.i'
     exodiff = 'bicubic_spline_interpolation_out.e'
     custom_cmp = 'bicubic_spline_interpolation.cmp'
-    group = 'x'
   [../]
   [./bicubic_spline_x]
     type = 'Exodiff'
     input = 'bicubic_spline_interpolation_x_normal.i'
     exodiff = 'bicubic_spline_interpolation_x_normal_out.e'
     custom_cmp = 'bicubic_spline_interpolation.cmp'
-    group = 'x'
   [../]
   [./bicubic_spline_y]
     type = 'Exodiff'
     input = 'bicubic_spline_interpolation_y_normal.i'
     exodiff = 'bicubic_spline_interpolation_y_normal_out.e'
     custom_cmp = 'bicubic_spline_interpolation.cmp'
-    group = 'x'
```

Comment:
You can already do this with the reg ex flag.

./run_tests --re='bicubic_spline_y'

Reply:
A little pedantic perhaps, but it would be great if you could squash these two commits